src/puppetlabs/puppetdb/scf/storage.clj

```diff
@@ -1159,6 +1159,14 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; Database support/deprecation
 
+(defn db-deprecated?
```

Comment:
@mullr hmm, I don't disagree in general, but I was just mirroring the db-unsupported? function right next to it.